src/sparsezoo/v2/test.md

```diff
@@ -0,0 +1,111 @@
+# API User pathway
+
+## Download one file
+
+### Through the string arguments
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "recipe=transfer_learn"
+# will read all the files from the server, except all the recipe files. The only recipe 
+# file that will be "seen" by the `model` is "transfer learn"
+model = Model(path=stub)
+model.download() # optional: may be called to download all the files, but is not required (see below). Will be omitted in further examples
+recipes: str = model.recipes.downloaded_path() # will automatically download the file (if not already downloaded) and return the path
```

Comment:
rewrite downloaded_path to path so it is more generic

additionally, recipes should be a parent folder that has a __dict__ implementation for individual recipe files underneath. Need an alias on the recipes object to a recipe file for the .default() recipe

Reply:
Coming to think about it, I'd rather make it:
`model.directory.get_path()` rather than `model.directory.path`

This is purely because of the fact, that when calling the `getter` in the latter case, sometimes we would need to go through the `download()` function. However, there is some internal logic in the codebase which is also calls `directory.path` on a few occasions. This situation requires introducing an argument (should I download on the call or not), clearly a sign that this should be a method, not a getter.

Reply:
@markurtz also, to have all the questions in one place:
What would be the use case of representing the recipe as a string --> str(recipe) ?

Reply:
(also model cards)